firmware/application/ui/ui_btngrid.cpp

```diff
@@ -116,6 +116,12 @@ void BtnGridView::add_items(std::initializer_list<GridItem> new_items) {
     update_items();
 }
 
+void BtnGridView::add_item(GridItem new_item) {
```

Comment:
Does the grid support scrolling if the number of items exceeds one screen? I've never tried.

Reply:
Yes it actually does. I fixed that recently.